src/main/java/org/sep3tools/PetroVisitor.java

```diff
@@ -6,7 +6,7 @@
 import java.util.logging.Logger;
 
 import static java.util.Objects.isNull;
-import static org.sep3tools.JavaConnector.getBodenQuant;
+import static org.sep3tools.JavaConnector.*;
```

Comment:
`*` import should be prevented.

Reply:
May be we need some code conventions and a tool which enforces those. Currently ... neither nor.